identity-service/src/tracer.js

```diff
@@ -0,0 +1,53 @@
+const {
+  trace,
+  diag,
+  DiagConsoleLogger,
+  DiagLogLevel
+} = require('@opentelemetry/api')
+
```

Comment:
any particular reason that this file is diff from `creator-node/src/tracer.js`?

Reply:
Besides RedisInstrumentation and PgInstrumentation being missing from identity, the files are almost identical. 

The `tracing` namespace is only in CN since I doubt we're going to be instrumenting Identity.